test/system/220-healthcheck.bats

```diff
@@ -273,4 +273,215 @@ Log[-1].Output   | \"Uh-oh on stdout!\\\nUh-oh on stderr!\\\n\"
     done
 }
 
+@test "podman healthcheck --health-max-log-count default value (5)" {
+    local repeat_count=10
+    local msg="Hello, How are you?"
+    local ctrname="c-h-$(safename)"
+    run_podman run -d --name $ctrname   \
+               --health-cmd "echo $msg" \
+               $IMAGE /home/podman/pause
+    cid="$output"
+
+    run_podman inspect $ctrname --format "{{.Config.HealthMaxLogCount}}"
+    is "$output" "5" "HealthMaxLogCount is set to 5"
+
+    for _ in $(seq 1 $repeat_count);
+    do
+        run_podman healthcheck run $ctrname
+        is "$output" "" "output from 'podman healthcheck run'"
```

Comment:
Suggestion: `"unexpected output from podman healthcheck run (pass $i)"` (and use `i` instead of `_`). Any time there's a loop, it's nice to include the variables in the error message. Not super helpful in this particular case, but it's a good habit to get into.